### ISSUE TITLE
Crash with handle NotificationMarkReadEvent

### ISSUE THREAD

**Issue by AlexanderKolesnikovSaritasa** [GetStream/stream-chat-android#536]
SDK 4.2.10-beta

When I open a screen with a channel, then I retrieve the next error and app crash also:

```
2020-06-19 09:48:32.160 22253-22382/com.witley E/AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-6
     Process: com.witley, PID: 22253
     kotlin.KotlinNullPointerException
         at io.getstream.chat.android.livedata.controller.ChannelControllerImpl.handleEvent(ChannelControllerImpl.kt:786)
         at io.getstream.chat.android.livedata.controller.ChannelControllerImpl.handleEvents(ChannelControllerImpl.kt:714)
         at io.getstream.chat.android.livedata.EventHandlerImpl.handleEventsInternal$livedata_release(EventHandlerImpl.kt:249)
         at io.getstream.chat.android.livedata.EventHandlerImpl$handleEventsInternal$1.invokeSuspend(Unknown Source:12)
         at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
         at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
         at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
         at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
         at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
         at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

It seems, that controller tries to handle some NotificationMarkReadEvent and there is not user field (null).

**Comment by tschellenbach** [comment]
I believe @samiuelson spotted this and has a fix

**Comment by samiuelson** [comment]
Released new artifact version`4.2.11-beta-5` with the fix
